### DISCUSSION TITLE
To apply the list of action operations for opa

### DISCUSSION THREAD

**Discussion by ElapsedSoul** [trinodb/trino#23379] [General]
For [opa](https://trino.io/docs/current/security/opa-access-control.html)
I need a list of action operations for opa to make rego file,
Or the team can give a rego demo with all kind of action operations would be more fine.

**Comment by sudohainguyen** [comment]
hey @ElapsedSoul I was in your situation before, official documentation look not decent though

you can refer to Bloomberg's doc [here](https://github.com/bloomberg/trino/blob/add-open-policy-agent-with-docs/plugin/trino-opa/schema-docs/README.md)

it does help me a lot too
